src/module.c

```diff
@@ -660,7 +660,6 @@ static int AlterIndexInternalCommand(RedisModuleCtx *ctx, RedisModuleString **ar
     return QueryError_ReplyAndClear(ctx, &status);
   }
 
-  FieldsGlobalStats_UpdateStats(sp->fields + (sp->numFields - 1), 1);
```

Comment:
Why removing this?

Reply:
Moved into the function that actually adds the fields